service-base/src/main/java/org/eclipse/hono/service/tenant/BaseTenantService.java

```diff
@@ -37,6 +23,18 @@
 import io.vertx.core.Future;
 import io.vertx.core.Handler;
 import io.vertx.core.json.JsonObject;
+import org.eclipse.hono.client.ClientErrorException;
+import org.eclipse.hono.client.ServerErrorException;
+import org.eclipse.hono.service.EventBusService;
+import org.eclipse.hono.tracing.TracingHelper;
+import org.eclipse.hono.util.EventBusMessage;
+import org.eclipse.hono.util.MessageHelper;
+import org.eclipse.hono.util.TenantConstants;
+import org.eclipse.hono.util.TenantResult;
+
+import javax.security.auth.x500.X500Principal;
+import java.net.HttpURLConnection;
+import java.util.Objects;
```

Comment:
this is the wrong import order, please use the order defined in `eclipse/hono.importorder`

Reply:
fixed.